.github/workflows/codeql-analysis.yml

```diff
@@ -9,36 +9,120 @@ name: CodeQL security analysis
 
 on:
   workflow_call:
-
 jobs:
+  analyzeExcludingWarnings:
+    name: Analyze add-on excluding warnings
```

Comment:
I can't seem to find where warnings are excluded, can you point to this?

Reply:
Warnings are excluded via a configuration file including filters to exclude warnings and recommendation. I read that this would show just important errors, and I checked that it's the case for the readFeeds add-on. Also MathCAT was rejected just for warnings.
Warnings are excluded in the first job, which will fail just for important errors.
If this job fails, the second one won't be run and the PR will be created.
If the first job doesn't fail, the second job will search for warnings (less important errors), and these will be reported to authors without creating the PR to add this to reviewedAddons.

Reply:
Where is this configuration file? can it be documented here please

Reply:
It's in the .github/codeql subfolder. It contains two configuration files, one to exclude warnings ant the other one to include them. The init action determines which configuration file should be used for each job.

Reply:
These files appear to have not been committed to this PR. 
The file on master doesn't contain anything relevant:
https://github.com/nvaccess/addon-datastore/blob/master/.github/codeql/codeql-config.yml

Reply:
Sorry, I tested all in my master branch and files were there.